relayer/cmd/run/beacon/command.go

```diff
@@ -62,7 +60,8 @@ func run(_ *cobra.Command, _ []string) error {
 
 	err = config.Validate()
 	if err != nil {
-		return fmt.Errorf("config file validation failed: %w", err)
```

Comment:
We only return errors from inner functions. This is the top level most function so we can safely log fatal here.